api/src/main/resources/marquez/db/migration/V47__add_lineage_event_indexes.sql

```diff
@@ -0,0 +1,5 @@
+CREATE INDEX lineage_events_event_time
+    on lineage_events(event_time DESC);
+
+CREATE INDEX lineage_events_namespace_event_time
+    on lineage_events(job_namespace, event_time DESC);
```

Comment:
Maybe do some perf test on a big db to measure the impact of this on POST /api/v1/lineage

Reply:
@wslulciuc @julienledem btw, we probably should do `CREATE INDEX CONCURRENTLY`